locales-pending/dashboard-premium.ftl

```diff
@@ -0,0 +1,173 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+premium-badge-label = { -brand-premium }
+premium-cta-label = Upgrade to { -brand-premium }
+
+# Variables:
+#   $limit (number) - Number of email addresses included in the plan
+modal-active-number-of-exposures-part-one-premium =
+  { $limit ->
+    [one] This chart includes the total number of times we found each type of data exposed across all data broker profiles and all data breaches for the { $limit } email address that you are currently monitoring.
+    *[other] This chart includes the total number of times we found each type of data exposed across all data broker profiles and all data breaches for up to { $limit } email addresses that you are currently monitoring.
+  }
+modal-active-number-of-exposures-part-three-premium = This chart does not include any exposures that are in-progress of being auto-removed. Once your exposures are fixed, they will be added to your total number of fixed exposures on the Fixed page.
+
+progress-card-auto-removed-headline = Auto-removed
+progress-card-in-progress-headline = In progress
+progress-card-locked-alt = Only available with { -brand-premium }
+
+progress-card-heres-what-we-fixed-headline-premium = Here’s what we fixed
+
+# Here’s What We Fixed Modal
+
+modal-heres-what-we-fixed-title = About what we fixed
+modal-heres-what-we-fixed-description-part-one = <b>Manually fixed</b> includes anything you’ve fixed yourself. All data breaches that require access to your accounts need
+  to be fixed manually, even if you’ve upgraded to { -brand-premium }.
+modal-heres-what-we-fixed-description-part-two = <b>Auto-removed</b> includes any exposures from data broker profiles that we’ve removed for you. This is available only for
+  { -brand-premium } subscribers.
+modal-heres-what-we-fixed-description-part-three = <b>In Progress</b> includes anything that we are currently
+  working on fixing. Removals typically take 7-14 days but the most
```

Comment:
```suggestion
  working on fixing. Removals typically take 7–14 days but the most
```

Reading a book on English style, I recently realized you should use _en dashes_ for intervals
https://www.thepunctuationguide.com/en-dash.html

Turns out it's also in our style guide: https://acorn.firefox.com/latest/content/punctuation-7J3JVM4o

Reply:
12ea98cb0